synapse/lib/stormlib/imap.py

```diff
@@ -507,21 +527,26 @@ def getObjLocals(self):
             'connect': self.connect,
         }
 
-    async def connect(self, host, port=imaplib.IMAP4_SSL_PORT, timeout=30, ssl=True, ssl_verify=True):
+    async def connect(self, host, port=imaplib.IMAP4_SSL_PORT, timeout=30, ssl=None):
 
         self.runt.confirm(('inet', 'imap', 'connect'))
 
-        ssl = await s_stormtypes.tobool(ssl)
+        ssl = await s_stormtypes.toprim(ssl)
         host = await s_stormtypes.tostr(host)
         port = await s_stormtypes.toint(port)
-        ssl_verify = await s_stormtypes.tobool(ssl_verify)
         timeout = await s_stormtypes.toint(timeout, noneok=True)
 
         ctx = None
+        hostname = None
         if ssl:
```

Comment:
I think this could just be `if ssl or port == imaplib.IMAP4_SSL_PORT:` and remove the elif block